index.html

```diff
@@ -0,0 +1,142 @@
+                </p></li>
+            </ul>
+            <ul class="program-container">
+                <li><img src="./Image/Icons/icon_02.png" alt="lecture-icon"></li>
+                <li><h2 class="program-header">CC Exhibition</h2></li>
+                <li><p class="program-text">Appreciate how spotify increase productivity and Microverse has unique way to learn
+                    remote pair programming and how it can benefit you and your organization.
+                </p></li>
+            </ul>
+            <ul class="program-container">
+                <li><img src="./Image/Icons/icon_03.png" alt="lecture-icon"></li>
+                <li><h2 class="program-header">Forum</h2></li>
+                <li><p class="program-text">This great time to share your thoughts
+                    and opinions with experts for each topic and summit comments/questions.
+                </p></li>
+            </ul>
+            <ul class="program-container">
+                <li><img src="./Image/Icons/icon_04.png" alt="lecture-icon"></li>
+                <li><h2 class="program-header">Workshop</h2></li>
+                <li><p class="program-text">Try to collaborate your own work using VS code
+                    rather than just watching. How fun pairing programming can be.           
+                </p></li>
+            </ul>
+            <ul class="program-container">
+                <li><img src="./Image/Icons/icon_04.png" alt="lecture-icon"></li>
+                <li><h2 class="program-header">CC Ignite</h2></li>
+                <li><p class="program-text">Get opportunities to network with cc affiliates
+                    around the world, also after the summit.
+                </p></li>
+            </ul>
+        </div>
+    </section>
+    <div class="join">
+        <h2 class="join-header">Join CC Global Pair Programming 2022</h2>
+        <h2 class="join-desktop">See the whole program</h2>
+    </div>
+    <section>
+        <h2 class="feature-speaker">Featured Speakers</h2>
+        <hr class="line-2">
+        <div id="modal"></div>
+    </section>
+    <footer id="footer">
+        <div class="more-footer">
+            <p class="more-p">More</p>
+            <img class="arrow-more" src="./Image/Icons/arrow_down.png" alt="arrow_down" >
+        </div>
+        <div class="partner-home">
+            <h2 class="partner-header">Partner</h2>
+            <hr class="line-1">
+            <div class="partner-image">
+                <img class="mozilla" src="./Image/Partner/mozilla.png" alt="mozilla-logo">
+                <img class="google" src="./Image/Partner/google-logo.png" alt="google-logo">
+                <img class="microverse" src="./Image/Partner/5e30ae0c4081806a8f50a7df_horizontal-logo-white.png" alt="Microverse-logo">
+                <img class="spotify" src="./Image/Partner/Spotify_Logo_CMYK_Green.png" alt="spotify-log">
+                <img class="lumen" src="./Image/Partner/lumen.png" alt="lumen-logo">
```

Comment:
* @aron-helu kindly fix all the broken images and icons on your website as i can see your project is missing the image folder you are trying to import from. below ⬇️ is a screenshot of how your webpage looks now 
![image](https://user-images.githubusercontent.com/52497246/176571453-170ef0ab-de77-4d57-94c6-88f39a683c14.png)